Src/BootCamp.Chapter/CleanTheFile.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    class CleanTheFile
+    {
+        public void Clean(string file, string outputFile)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'Clean' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)